File: BehaviorPattern/Command/Golang/Invokers/RemoteControl.go

package invokers

import (
	"v/Interfaces"
)

/**
 * Class Remote Control
 */
type RemoteControl struct {
    command interfaces.Command
}

// Set command
func (r *RemoteControl) SetCommand(command interfaces.Command) {
	r.command = command
}

// Execute command
func (r *RemoteControl) PressButton() {
	r.command.Execute()
}

File: BehaviorPattern/Command/Python/Invokers/RemoteControl.py

from Interfaces import Command

class RemoteControl:
    """
        Class Remote Control
    """
    def __init__(self):
        self.command: Command = None
        pass
    
    def setCommand(self, command: Command):
        """
            Set command
        """
        self.command = command
        
    def pressButton(self):
        """
            Press button
        """
        self.command.execute()